.github/dependabot.yml

```diff
@@ -67,19 +72,13 @@ updates:
           - "unified"
       eslint:
         patterns:
-          - "eslint"
-          - "eslint-*"
-          - "@typescript-eslint/*"
-          - "@eslint-react/*"
+          - "*eslint*"
       emotion:
         patterns:
-          - "@emotion/*"
-          - "@emotion-icons/*"
+          - "*emotion*"
       vitest:
         patterns:
-          - "vitest"
-          - "vitest-*"
-          - "@vitest/*"
+          - "*vitest*"
```

Comment:
Drive-by simplification to include all eslint / emotion / vitest dependencies in the respective groups.